### DISCUSSION TITLE
Is @28 pinned to git version? What's the way to upgrade if not?

### DISCUSSION THREAD

**Discussion by memeplex** [d12frosted/homebrew-emacs-plus#400] [Q&A] [answered]
Hi all,

I'm not too well versed in brew yet, I've installed the @28 version of emacs plus and now I want to upgrade to the latest revision of emacs. There are two versions to be considered here, the one for the formula and the one for emacs itself. From the point of view of brew, it seems that nothing has changed, so should I force an upgrade somehow? Will that work? What's the recommended way to do it? All this supposing emacs-plus@28 is not pinning emacs to a particular version, of course, which I think it's the case after a very cursory glance at the formula.

Thanks,
Carlos

**Comment by memeplex** [comment, marked as answer]
Sorry, I didn't realize it was already answered before (https://github.com/d12frosted/homebrew-emacs-plus/discussions/354). If you have a way to delete this post altogether please do it, I don't see how to do it myself.

**Reply by d12frosted** [reply]
No worries. It's really confusing, because 28 is still in development and you have literally to install it from scratch. Glad you figured that out :)
